0-ad-history-mod-package/simulation/templates/units/ptol/siege_tower.xml

```diff
@@ -4,6 +4,13 @@
     <Civ>ptol</Civ>
     <SpecificName>Helépolis</SpecificName>
     <Icon>units/hele/siege_tower.png</Icon>
+    <History>
+        Probably the most famous siege tower in Ancient Greece was the Helepolis ("Taker of Cities"). It was invented by Polyidus of Thessaly for the Macedonian army under Philip II in the 4th century B.C. and later improved by Epimachus of Athens for Demetrius I ("The Besieger").
+        The Helepolis was by far the biggest, most fearsome, and intimidating war machine in the ancient world. It was more than 100 feet tall and more than 50 feet wide. On the outside the walls were covered in metal plates and wet sheets of leather to prevent it from catching on fire. Only the back side facing away from the defenders was open. On the inside, the Helepolis had up to nine floors connected with ladders. 
+        It carried many stone, bolt, and arrow-launching devices that fired through small holes in the wall. Because the whole tower weighed about 160 tons, 3400 men working in relays were required to propel the tower on its eight wheels. Once the siege tower was close to the walls, it could unfold a bridge, leading the soldiers inside the tower directly onto the enemy's wall. 
```

Comment:
```suggestion
        It carried many stone, bolt, and arrow-launching devices that fired through small holes in the wall. Because the whole tower weighed about 160 tons, 3400 men working in relays were required to propel the tower on its eight wheels. Once the siege tower was close to the walls, it could unfold a bridge, leading the soldiers inside the tower directly onto the enemy's wall.
```